### DISCUSSION TITLE
[Question] Status of iOS support

### DISCUSSION THREAD

**Discussion by rocketraman** [dropbox/Store#666] [General]
Hello, store5 looks like exactly soemthing I would need for a use case I have for a new project. However, while researching the tool, I saw [this comment](https://www.reddit.com/r/androiddev/comments/13mb2sz/comment/jll24dq/) on Reddit:

![image](https://github.com/user-attachments/assets/d79b307a-5c0c-45b7-9bbd-596e62696889)

Since that comment was about a year ago and there have been 5.1.0 alpha releases since then (but I note no betas or releases), what would the current status of store5 on iOS be?

**Comment by lukisk** [comment]
Hi, we have been using the Store in iOS and android applications (worldwide 100M+ users) for over a year and haven't encountered any issues related to the Store. We use Store + Apollo in KMP for caching, fetching, and updating data, and in iOS, the flow from the Store is used in TCA reducers.
